src/main/java/com/github/wovnio/wovnjava/HtmlMarkerManager.java

```diff
@@ -0,0 +1,24 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.HashMap;
+
+import org.jsoup.nodes.Element;
+
+class HtmlMarkerManager {
```

Comment:
I would recommend just straight-up copying the HtmlReplaceMarker implementation from the other backends.

Reply:
wovnjava notably does not support the "attribute markers" (addValue in other backends), which would not be supported by this implementation as-is.

Reply:
Actually, it does _attempt_ to do this in `replaceForm`, but it just replaces the whole `<input type="hidden">` node itself with a comment. Other backends just replace the value attribute.

Reply:
Got it.